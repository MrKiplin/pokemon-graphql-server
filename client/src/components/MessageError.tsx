import React from "react";
import { Message } from "semantic-ui-react";

interface MessageErrorProps {
  message: string;
}

const MessageError: React.FC<MessageErrorProps> = ({ message }) => (
  <Message negative>
    <Message.Header>Error</Message.Header>
    <p>{message}</p>
  </Message>
);

export default MessageError;
