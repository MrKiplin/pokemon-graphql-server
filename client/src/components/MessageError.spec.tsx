import { mount } from "enzyme";
import MessageError from "./MessageError";

describe("MessageError", () => {
  it("Should match props and snapshot", () => {
    const component = mount(<MessageError {...{ message: "test-message" }} />);

    expect(component).toMatchSnapshot();
    expect(component.find("MessageError").prop("message")).toEqual(
      "test-message"
    );
  });
});
