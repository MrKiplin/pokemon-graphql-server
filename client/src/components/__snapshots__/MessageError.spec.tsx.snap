// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageError Should match props and snapshot 1`] = `ReactWrapper {}`;
